torch/_inductor/compile_fx.py

```diff
@@ -729,11 +755,17 @@ def fw_compiler_freezing(
     graph_id,
     forward_device,
 ):
-    from torch._inductor.freezing import freeze
+    from torch._inductor.freezing import convert_conv_weights_to_channels_last, freeze
 
     # partition_fn won't be called
     joint_graph_passes(aot_autograd_model)
 
+    layout_opt = GraphLowering.decide_layout_opt(aot_autograd_model)
+    if layout_opt:
+        # make sure meta['val'] is properly setup
+        fake_tensor_prop(aot_autograd_model, aot_example_inputs, True)
+        convert_conv_weights_to_channels_last(aot_autograd_model)
```

Comment:
I would have thought we needed the fake_tensor_prop after, but not before. I do think we need it after or the meta's will be incorrect

Reply:
Run a FakeTensorProp before the conversion to make sure meta['val'] is properly setup. I'm worried that they may not already be setup.

But if we are sure FakeTensorProp has already been called before, we can save this call.

Reply:
I think we should add it again, since now the existing meta's will be incorrect, since we've changed striding